docs/interactive/launchbuttons.ipynb

```diff
@@ -19,7 +19,24 @@
     "  url                       : https://github.com/yourusername/yourbookrepo  # Online location of your book\n",
     "  path_to_book              : path/to/book  # Optional path to your book, relative to the repository root\n",
     "  branch                    : master  # Which branch of the repository should be used when creating links (optional)\n",
-    "```"
+    "```\n",
+    "The launch button will use [nbgitpuller](https://github.com/jupyterhub/nbgitpuller) to pull your\n",
+    "repository to the chosen Binder, Google Colab, or JupyterHub location.  The `repository:`\n",
+    "configuration line uses `url`, `path_to_book`, and `branch` to find your notebook files _both MyST\n",
+    "Markdown and *.ipynb_.  \n",
+    "\n",
+    "- The `url` should be a GitHub repository that includes all of your files used to build the\n",
+    "  Jupyter-book. Your repository should include:\n",
+    "    - `_build`\n",
+    "    - `_config.yml`\n",
+    "    - `conf.py`\n",
+    "    - `requirements.txt`\n",
+    "    - _any other *.ipynb and *.md files for the book_\n",
+    "- The `path_to_book` should be left blank if the repository is hosting the Jupyter-book. If you are\n",
```

Comment:
Could we just use a sentence like:

"`path_to_book` should be used if your book is in a sub-folder of your repository (e.g., `docs/` or `book/`)."

Reply:
Thanks, I like that. I'll update.